noir-projects/aztec-nr/aztec/src/macros/functions/mod.nr

```diff
@@ -22,8 +22,8 @@ use interfaces::{create_fn_abi_export, register_stub, stub_fn};
 // Therefore, the execution order of `#[internal]` and `#[private]` is irrelevant.
 
 /// Internal functions can only be called by the contract itself, typically from private into public.
-pub comptime fn internal(_f: FunctionDefinition) {
-    // Marker attribute
+pub comptime fn internal(f: FunctionDefinition) {
+    f.add_attribute("internal")
 }
```

Comment:
I don't understand, doesn't this depend on the order of execution of the macros? There's a big comment right above this function that goes over what I called 'marker attributes' and how they solve the unknown execution order issue.